src/unlink.js

```diff
@@ -1,15 +1,13 @@
 const path = require('path');
 const log = require('npmlog');
 
-const isEmpty = require('./isEmpty');
 const getProjectDependencies = require('./getProjectDependencies');
 const unregisterDependencyAndroid = require('./android/unregisterNativeModule');
 const unregisterDependencyIOS = require('./ios/unregisterNativeModule');
 const unlinkAssetsAndroid = require('./android/unlinkAssets');
 const unlinkAssetsIOS = require('./ios/unlinkAssets');
 const getDependencyConfig = require('./getDependencyConfig');
-const diff = require('./diff');
-const flatMap = require('./flatMap');
+const _ = require('lodash');
 
```

Comment:
Isn't it better to require each function separately? Just to make it clearer what do we use (that's mainly a personal preference)